src/app/components/activity/feed/event-item/event-item.ts

```diff
@@ -164,14 +150,18 @@ export default class AppActivityFeedEventItem extends Vue {
 		return this.$router.resolve(this.link).href;
 	}
 
-	get shouldShowFollow() {
+	get shouldShowFollowInHeader() {
 		// Don't show follow for game posts. Only for user posts/videos.
-		return (
-			this.feed.shouldShowFollow &&
-			!(this.post && this.post.game) &&
-			this.user &&
-			!this.user.is_following
-		);
+		if (!this.feed.shouldShowFollowInHeader || !this.post || this.post.game) {
```

Comment:
You say videos, but then check if post exists. For comment videos, won't `this.post` be null?

Reply:
I didnt touch the comment, i just split that single return into a more readable form haha.

I guess the original intention was to support follows on comment videos but that doesnt seem implemented hmm. For now i'll remove the bit about videos I guess.

Reply:
I mean the code is wrong, not the comment. Shouldn't it be this?
`if (!this.feed.shouldShowFollowInHeader || (this.post && this.post.game)) {`